middleware/index.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const path = require('path');
+const fs = require('fs');
 
-module.exports = ({ middlewarePath = '/falcor-postman', app }) => {
+module.exports = ({ middlewarePath = '/falcor-postman', falcorModelPath = '/model.json', app, fileSystem = fs }) => {
```

Comment:
I am not entirely sure about this; now options has got also `fileSystem` because of `[HMR]` stuff; what do you think about this @nickbalestra ?

// cc @chriscartlidge